package.json

```diff
@@ -265,18 +266,29 @@
     }
   },
   "devDependencies": {
+    "@babel/core": "^7.4.4",
```

Comment:
are there any cons of having a separate package.json for storybook? I'm thinking about people not familiar with brave-core wanting to add/remove deps in storybook and potentially removing something we need for other feature. I think it's unlikely that others would `npm run build` when working solely on storybook

Reply:
That's valid, but we'd have to weigh the benefits of that vs increase of developer overhead. How do you see it working? A storybook sub-project that the front-end developer must go to to run storybook. I wonder how that will work when webpack / ts in storybook sub-project it's building JS / TS output of features in parent project...

Reply:
I created a [doc](https://docs.google.com/document/d/17jlrwx31iHad-Qt3rp6i6hleDw9jdmxvmb4CBQA0r8M/edit) about things we like to consider including our first convo re locale strings. I'm ok with it in the same package.json until we better evaluate

Reply:
@cezaraugusto can we just use the [existing](https://docs.google.com/document/d/11vCLLDJ11DdjJ92M9TDgVpqcdHFw6j82VH-qE3Y_vYc/edit#heading=h.96za1bea8dfn) doc?